docs/administration/configuring-replication/create-replication-rules.md

```diff
@@ -47,7 +47,11 @@ A replication endpoint must exist before you create a replication rule. To creat
 
    If you do not enter a namespace, resources are placed in the same namespace as in the source registry. 
 
-   ![Destination and namespaces](../../../img/replication-rule4.png)
+   ![Destination and namespaces](../../img/replication-rule4.png)
```

Comment:
../../../ should be correct, it should be one more level than the actual folder structure, the image has its own directory basically, please double check